plugin.video.zdf_de_2016/addon.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.zdf_de_2016" name="ZDF Mediathek 2016" version="1.0.11" provider-name="Generia">
+<addon id="plugin.video.zdf_de_2016" name="ZDF Mediathek 2016" version="1.0.12" provider-name="Generia">
   <requires>
     <import addon="xbmc.python" version="2.14.0"/>
```

Comment:
Please fix `WARN: For jarvis it is advised to set xbmc.python version to 2.24.0` with the next update.